packages/backend/src/graphql/schema.graphql

```diff
@@ -198,17 +198,17 @@ type Mutation {
     input: WithdrawEventLiquidityInput!
   ): LiquidityMutationResponse
     @deprecated(
-      reason: "Use `withdrawOutgoingPaymentLiquidity, withdrawIncomingPaymentLiquidity, or createWalletAddressWithdrawal`"
+      reason: "Use `createOutgoingPaymentWithdrawal, createIncomingPaymentWithdrawal, or createWalletAddressWithdrawal`"
     )
 
   "Withdraw incoming payment liquidity"
-  withdrawIncomingPaymentLiquidity(
-    input: WithdrawIncomingPaymentLiquidityInput!
+  createIncomingPaymentWithdrawal(
```

Comment:
Do we want to rename these? If so, would we want to do the same with the deposit mutations?

I'm understanding that you renamed these to match `createWalletAddressWithdrawal` ?

CC @BlairCurrey, since you were working on deprecating event liquidity in favour of these mutations

Reply:
Here is the discussion we had on that: https://github.com/interledger/rafiki/pull/2151#discussion_r1406673266

The finding was that the `create[RESOURCE]Withdrawal` denotes a 2 phase transfer whereas a `withdraw[RESOURCE]Liquidity` denotes a single phase ([see Sabine's comment](https://github.com/interledger/rafiki/pull/2151#discussion_r1413600059)). So I think the way Jason renamed this is good. And that we dont need to renamed the deposit mutations since because is no 1 vs. 2 phase, or they're all just 1 phase... correct me if I'm wrong there.